apps/dg/english.lproj/strings.js

```diff
@@ -629,6 +629,7 @@ SC.stringsFor('English', {
   'DG.Inspector.newAttribute': "New Attribute in %@...",  // "New Attribute in %@..."
   'DG.Inspector.randomizeAllAttributes': "Rerandomize All", // "Randomize Attributes"
   'DG.Inspector.exportCaseData': "Export Case Data...", // "Export Case Data..."
+  'DG.Inspector.exportPrimaryCaseData.toolTip': "Export data from this case", // "Export Case Data..."
```

Comment:
After review I figured Jonathan would handle merging it. I apparently didn't assign it to him in Github, however. I'll check with him to see how he wants to handle it.